integrated_channels/canvas/client.py

```diff
@@ -78,22 +91,62 @@ def course_update_endpoint(canvas_base_url, course_id):
         )
 
     def create_content_metadata(self, serialized_data):
+        """
+        Creates a course in Canvas.
+        If course is not found ,easy!  create it as usual
```

Comment:
nit, I don't mind the `easy!` being there, but comma placement

Reply:
It boooooothers :P